### PR TITLE
Add better usergrid

### LINKED ISSUE #633: Make RegisteredCell a generic component
Move `app/routes/events/components/RegisteredCell.js` from events into `app/components`.

An example for the styleguide should also be added.

Mentioned in #631. 😄

### PR DESCRIPTION
**Pull request by odinuge**
This fixes #633.

Grid (aka. css-grid) is awesome!

### CODE REVIEW

File: app/routes/events/components/EventDetail/index.js [hunk 1]
```diff
@@ -1,11 +1,10 @@
 // @flow
 
 import styles from './EventDetail.css';
-import React, { Component } from 'react';
+import * as React from 'react';
```
Comment (SmithPeder) [inline app/routes/events/components/EventDetail/index.js]: plz React, { Component, Fragment }

File: app/routes/events/components/EventDetail/index.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ type Props = {
 /**
  *
  */
-export default class EventDetail extends Component<Props> {
+export default class EventDetail extends React.Component<Props> {
```
Comment (SmithPeder) [inline app/routes/events/components/EventDetail/index.js]: extends Fragment

Comment (odinuge) [reply app/routes/events/components/EventDetail/index.js]: :sob:

Comment (odinuge) [reply app/routes/events/components/EventDetail/index.js]: screenshot from 2018-10-19 18-54-28
plz make up your mind

**Pull request merged**